gnomad_constraint/pipeline/constraint_pipeline.py

```diff
@@ -13,15 +13,20 @@
 from gnomad_constraint.resources.resource_utils import (
     get_sites_resource,
     get_preprocessed_ht,
+    get_training_dataset,
     get_logging_path,
     annotated_context_ht,
+    mutation_rate_ht,
     CURRENT_VERSION,
     VERSIONS,
     DATA_TYPES,
+    GENOMIC_REGIONS,
+    POPS,
```

Comment:
I saw examples in gnomad_method that imports functions first and then constants, so I guess the order is fine.

Reply:
correct, we didn't always follow the import ordering that we do now, and never went back to update it. See https://github.com/broadinstitute/gnomad_methods/pull/490 I think we should just use isort so it's all consistent and we don't need to think about ordering on each PR